### DISCUSSION TITLE
How to change path folder?

### DISCUSSION THREAD

**Discussion by galeka** [Dineshkarthik/telegram_media_downloader#108] [Q&A] [answered]
Hi,

I've tried and it works fine, great! 
Is it possible to change the path folder?

**Comment by Dineshkarthik** [comment, marked as answer]
Of course,  you can.
Change the `THIS_DIR` variable to the absolute path of the folder location in line [#L112](https://github.com/Dineshkarthik/telegram_media_downloader/blob/master/media_downloader.py#L112) and [#L120](https://github.com/Dineshkarthik/telegram_media_downloader/blob/master/media_downloader.py#L120) in `media_downloader.py`

Ex:

```py
# L112
file_name: str = os.path.join(
    "/home/path/to/folder",
    _type,
    "voice_{}.{}".format(dt.utcfromtimestamp(media_obj.date).isoformat(), file_format),
)
```

```py
# L120
file_name = os.path.join(
    "/home/path/to/folder", _type, getattr(media_obj, "file_name", None) or ""
)
```

**Reply by rmdes** [reply]
Would be really neat to add this to the config.yml so that we don't have to touch the code?

**Reply by rmdes** [reply]
Like in this fork ? https://github.com/Codel1417/telegram_media_downloader

**Reply by Dineshkarthik** [reply]
@rmdes here is the [contribution guide](https://github.com/Dineshkarthik/telegram_media_downloader/blob/master/CONTRIBUTING.md) you are more than welcome to create a PR for this.

**Reply by rmdes** [reply]
Sadly not a coder, just starting to scratch the surface of Python :/
but I can contribute with testing and actually a very original project I'm running i'll be glad to expose to you if you're interested :) (related to Ukraine)
